test/jdk/java/awt/event/ComponentEvent/TextComponentTextEventTest.java

```diff
@@ -112,21 +110,20 @@ public static void main(String[] args) throws Exception {
                 robot.waitForIdle();
                 if (textComp instanceof TextField && textChanged) {
                     throw new RuntimeException(
-                        "FAIL: TextEvent triggered when Enter pressed on in "
-                            + textComp);
+                        "FAIL: TextEvent triggered when Enter pressed on "+ textComp);
```

Comment:
```suggestion
                        "FAIL: TextEvent triggered when Enter pressed on " + textComp);
```

Reply:
It's not resolved: there's still no space before `+` operator.

And you removed it from other places for some reason where they were present.